sdk/examples/intkey_rust/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+extern crate cbor;
+extern crate crypto;
+#[macro_use]
+extern crate log;
+extern crate sawtooth_sdk;
+
+mod handler;
+
+use sawtooth_sdk::processor::TransactionProcessor;
+
+use handler::IntkeyTransactionHandler;
+
+pub fn start(endpoint: &str) {
+    let handler = IntkeyTransactionHandler::new();
+    let mut processor = TransactionProcessor::new(endpoint);
+    processor.add_handler(&handler);
+    processor.start();
+}
```

Comment:
I understand why you need to create a lib.rs file in intkey rust. I think it makes more sense from the perspective of intkey rust that the library only publish the IntkeyTransactionHandler and that the test handle starting up the processor.